docs/Installation.md

```diff
@@ -1,33 +1,6 @@
 # Installation
 
-## Quickstart
```

Comment:
Basically switched up the order of information here to follow the order someone getting setup for the first time might follow. Still not sure if this is the best flow.